Libplanet/Net/Messages/Message.cs

```diff
@@ -21,7 +21,7 @@ internal enum MessageType : byte
             /// <summary>
             /// A reply to <see cref="Ping"/>.
             /// </summary>
-            Pong = 0x14,
+            Pong = 0x15,
```

Comment:
Why do we need to bump this type id? If I remember correctly, we've never released `0x14` yet.